Source/lua/modules/items.cpp

```diff
@@ -114,47 +114,353 @@ void InitItemUserType(sol::state_view &lua)
 
 void RegisterItemTypeEnum(sol::state_view &lua)
 {
-	lua.new_enum<ItemType>("ItemType", { { "Misc", ItemType::Misc }, { "Sword", ItemType::Sword }, { "Axe", ItemType::Axe }, { "Bow", ItemType::Bow }, { "Mace", ItemType::Mace }, { "Shield", ItemType::Shield }, { "LightArmor", ItemType::LightArmor }, { "Helm", ItemType::Helm }, { "MediumArmor", ItemType::MediumArmor }, { "HeavyArmor", ItemType::HeavyArmor }, { "Staff", ItemType::Staff }, { "Gold", ItemType::Gold }, { "Ring", ItemType::Ring }, { "Amulet", ItemType::Amulet }, { "None", ItemType::None } });
+	// clang-format off
+	lua.new_enum<ItemType>("ItemType", {
+		{ "Misc", ItemType::Misc },
+		{ "Sword", ItemType::Sword },
+		{ "Axe", ItemType::Axe },
+		{ "Bow", ItemType::Bow },
+		{ "Mace", ItemType::Mace },
+		{ "Shield", ItemType::Shield },
+		{ "LightArmor", ItemType::LightArmor },
+		{ "Helm", ItemType::Helm },
+		{ "MediumArmor", ItemType::MediumArmor },
+		{ "HeavyArmor", ItemType::HeavyArmor },
+		{ "Staff", ItemType::Staff },
+		{ "Gold", ItemType::Gold },
+		{ "Ring", ItemType::Ring },
+		{ "Amulet", ItemType::Amulet },
+		{ "None", ItemType::None }

```

Comment:
If you add a trailing comma after the last item, I think clang-format will format it the right way

Reply:
Clang-format did a yikes that time with trailing comma. Should we keep this last commit?

Reply:
I think the indentation may be made more reasonable by adding a newline before the initialiser list

Reply:
Thank you for your patience!